pages/app-tours.vue

```diff
@@ -10,7 +10,7 @@
         </h1>
         <p>
           We leverage <a href="https://shepherdjs.dev/">Shepherd.js</a> to create step-by-step application tours and feature
-          introductions to help new users get up to speed quickly with your web app.
+          introductions to help new users get up to speed quickly with your web application.
```

Comment:
@chuckcarpenter I think we should leave this as `app` so we say both application and app, rather than just application.